mitiq/utils.py

```diff
@@ -361,3 +361,28 @@ def operator_ptm_vector_rep(opt: NDArray[Any]) -> NDArray[Any]:
             np.trace(opt @ kron_product) * np.sqrt(1 / 2**num_qubits)
         )
     return np.array(opt_vec)
+
+
+def qem_methods() -> Dict[str, str]:
```

Comment:
I'm not sure them meaning of "depth" would necessarily be clear in this context, as it might be confused with circuit depth, as a kwarg for a particular QEM technique. Do you think the docstring needs to be clearer about how you can access just the module names (i.e. explicitly writing out`qem_modules().keys()`?)